src/main/frontend/app/containers/ReviewFormContainer.js

```diff
@@ -0,0 +1,79 @@
+import React, {Component} from 'react'
+import InputField from '../components/InputField'
+
+class ReviewFormContainer extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      rating: '',
+      comment: ''
+    }
+
+    this.handleRatingChange = this.handleRatingChange.bind(this)
+    this.handleCommentChange = this.handleCommentChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.handleFormClear = this.handleFormClear.bind(this)
+  }
+
+handleRatingChange(event) {
+  this.setState( { rating: event.target.value} )
+}
+
+handleCommentChange(event) {
+  this.setState( { comment: event.target.value} )
+}
+
+handleFormClear(event) {
+  this.setState( {
+    rating: '',
+    comment: ''
+  } )
+}
+
+handleSubmit(event) {
+  event.preventDefault()
+  let hasErrors = false
+  if(this.state.comment.trim() === "") {
+    alert("Please enter a comment.")
```

Comment:
not wild about showing an alert - can we do something a little more interactive in page?